platform/search/bundle.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Search",
+    "description": "Allows the user to search through the file tree.",
+    "extensions": {
+        "constants": [
+            {
+                "key": "GENERIC_SEARCH_ROOTS",
```

Comment:
@VWoeltjen @slhale As a platform feature, I think generic search should be opt-out, not opt-in.  Can we do "GENERIC_SEARCH_EXCLUDE_ROOTS" instead?